domain/dagconfig/params.go

```diff
@@ -147,6 +147,8 @@ type Params struct {
 
 	// DisableDifficultyAdjustment determine whether to use difficulty
 	DisableDifficultyAdjustment bool
+
+	SkipProofOfWork bool
```

Comment:
Please add a test that makes sure that SkipProofOfWork is false in {main,dev,test,sim}net, so that nobody flips it by mistake.